### PR TITLE
ObsRsync: Include URL in GRU task

### PR DESCRIPTION
**Pull request by kalikiana**
This fascilitates debugging of issues with sync.

See: https://progress.opensuse.org/issues/112871

### CODE REVIEW

**Review by perlpunk — APPROVED** [review]

**Comment by okurz** [comment]
t/ui/27-plugin_obs_rsync_obs_status.t failed. I suggest you run that locally

File: lib/OpenQA/WebAPI/Plugin/ObsRsync/Controller/Gru.pm [hunk 1]
```diff
@@ -128,7 +128,7 @@ sub run {
     return $self->render(json => {message => 'queue full'}, status => QUEUE_FULL)
       if ($results->{total} >= $queue_limit);
 
-    $app->gru->enqueue('obs_rsync_run', {project => $project}, {priority => 100});
+    $app->gru->enqueue('obs_rsync_run', {project => $project, url => $helper->project_status_url}, {priority => 100});
```
Comment (kalikiana) [inline lib/OpenQA/WebAPI/Plugin/ObsRsync/Controller/Gru.pm:131]: I can't make sense of the unit test failure. Adding the `url` argument somehow causes the list of inactive projects to be different. Changing `_extend_job_info` doesn't seem relevant. There must be something else?

Comment (perlpunk) [reply lib/OpenQA/WebAPI/Plugin/ObsRsync/Controller/Gru.pm:131]: The adjustment needs to be done here as well: https://github.com/os-autoinst/openQA/blob/master/lib/OpenQA/WebAPI/Plugin/ObsRsync/Controller/Folders.pm#L75
```
            next unless (ref $args eq 'HASH' && scalar(%$args) == 1 && $args->{project});
```

Comment (kalikiana) [reply lib/OpenQA/WebAPI/Plugin/ObsRsync/Controller/Gru.pm:131]: That was it! Thank you so much!

**Pull request merged**